packages/amplication-git-pull-request-service/src/core/diff/visitors/mocks.ts

```diff
@@ -0,0 +1,11 @@
+import { Difference } from 'dir-compare';
+import { mock } from 'jest-mock-extended';
+
+export const name1PlaceHolder = 'name1';
+
+export const missingFolderMock = mock<Difference>({
+  name1: name1PlaceHolder,
```

Comment:
```suggestion
export const missingFolderMock = mock<Difference>({
  name1: 'name1',
```
Can be a lot simpler :)

Reply:
Yes, but @yuval-hazaz wants us to put consts in outside vars, @yuval-hazaz Please approve / disaprrove

Reply:
Let's not let this be a blocker. If we can move faster, I am okay with keeping it as is.

Reply:
I understand, but you are not making use of this exported variable at all ([check here](https://github.com/amplication/amplication/blob/80609f1df85a9cff5198500ad452f994ffe78038/packages/amplication-git-pull-request-service/src/core/diff/visitors/delete-files.spec.ts#L19))

So it has exactly the same meaning if you put it inside your `mock()` function, which contains a bunch of other consts anyway.